affirm/src/main/java/com/affirm/android/GetCardRequest.java

```diff
@@ -0,0 +1,129 @@
+package com.affirm.android;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import androidx.annotation.NonNull;
+
+import com.affirm.android.exception.APIException;
+import com.affirm.android.exception.AffirmException;
+import com.affirm.android.exception.ConnectionException;
+import com.affirm.android.model.CardDetails;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import static com.affirm.android.AffirmConstants.GET_CHECKOUT_CARD_PATH;
+import static com.affirm.android.AffirmTracker.TrackingEvent.NETWORK_ERROR;
+import static com.affirm.android.AffirmTracker.TrackingLevel.ERROR;
+import static com.affirm.android.AffirmTracker.createTrackingNetworkJsonObj;
+
+public class GetCardRequest implements AffirmRequest {
+
+    @NonNull
+    private final String checkoutId;
+
+    @NonNull
+    private final GetCardRequestCallback callback;
+
+    private Call call;
+
+    public GetCardRequest(@NonNull String checkoutId,
+                          @NonNull GetCardRequestCallback callback) {
+        this.checkoutId = checkoutId;
```

Comment:
I'm curious to know what auth allows us to get the VCN data? is is just by checkoutId?